nbs/15_gsa_gls/20-null_simulations/05_gwas/04_10-qc-variants-hardy-plot.ipynb

```diff
@@ -0,0 +1,860 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    head(hwe)
</pre><p>is this a p-value? It might be worth being a bit more clear about what it in the plot title!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Yes, these are p-values. Added a comment to explain better! It's a good idea, I will definitely forget this part in a few months haha.</p>